### DISCUSSION TITLE
Support for RTL Languages!

### DISCUSSION THREAD

**Discussion by Crazyrzy** [Chatterino/chatterino2#3794] [Ideas] [answered]
Good evening, I was wondering when will RTL (Right to Left, Arabic, Hebrew, Iranian) languages have support on the application.

Currently, the messages appear in reverse. 

![image](https://user-images.githubusercontent.com/41770712/171932832-f82c3891-3a3c-40f4-a0de-55f838c78027.png)
![image](https://user-images.githubusercontent.com/41770712/171932857-472c30a1-9c0f-498b-acda-f392da9ee287.png)

First Image, correct way of lining up.
Second Image, on Chatterino.

For further clarification, if I happened to say in Arabic: "Good evening!" it would come out as "evening! Good"

**Comment by iProdigy** [comment]
Duplicate of #720

**Comment by Felanbird** [comment, marked as answer]
@Crazyrzy Proper RTL support was added in #3958 - I attempted to @ everybody who ever asked about this feature, in #720, but it appears I missed those who opened discussions.
